packages/serialized-state-manager/src/internal/client-side-state-manager.ts

```diff
@@ -0,0 +1,14 @@
+export class ClientSideStateManager {
+  private serializedStateByConsumerUid?: Record<string, string | undefined>;
```

Comment:
rename to `serializedStatesByConsumerUid`

Reply:
fixed